assets/js/common/Table/Table.jsx

```diff
@@ -189,7 +190,7 @@ function Table({
             className={classNames(
               'min-w-fit shadow overflow-hidden rounded-b-lg',
               {
-                'rounded-t-lg': !hasFilters,
+                'rounded-t-lg': !hasFilters && roundedTop,
```

Comment:
With this every Table will have now the roundedTop :) 
I guess you don't mind right?
I checked it looks good :+1:

Reply:
Hey Eugen, the additional boolean here `roundedTop` helps yo override a scenario where a table has no filters and was previously displaying rounded corners. This was the case after rearranging the HTML elements for the **Dashboard**. Also there should be no impact for current tables but rather a new way for us to override the behaviour if we want the table to display with top rounded corners since this is the default behaviour. 😊 

Here is an example without the boolean:
<img width="409" alt="Screenshot 2025-01-22 at 12 42 16" src="https://github.com/user-attachments/assets/c8f6f49a-126c-48ee-a367-42536ec612b1" />

Reply:
Yeah, all good, i think i was not clear enough what i meant.  The default behavior won't change. 
You are right we now have the option to control it with `roundedTop = true` 
All fine here :)